sb_vision/distance_finding.py

```diff
@@ -0,0 +1,121 @@
+"""
+Handlers for detecting 3D marker Location.
+
+Also handles loading the calibration file which is passed into the
+location finding function.
+"""
+
+import functools
+import re
+from pathlib import Path
+from typing import Any, Dict, List, NamedTuple, Tuple
+
+import cv2
+import numpy as np
+from lxml import etree
+
+Coordinate = NamedTuple('PixelCoordinate', [('x', float), ('y', float)])
+
+
+def _get_values_from_xml_element(element: etree.Element) -> List[str]:
+    """Parse an xml tag with space-separated variables."""
+    text = []  # type: List[str]
+    for e in element.itertext():
+        if e.strip():
+            text += re.split('\s+', re.sub('\n', '', e.strip()))
+    return text
+
+
+def get_calibration(file_name: Path) -> Dict[str, Any]:
```

Comment:
This function contains conversions for a bunch of things we're not using. Specifically we need the keys `'cameraMatrix'` `'dist_coeffs'` yet the code also seems to cope with translating a bunch of others.

Are you expecting that we'll need those other things at some point, or could we drop the redundant code?

Note: this may make some of the following review points redundant.